clientlibs/js/webpack.config.ts

```diff
@@ -1,6 +1,6 @@
-import path = require('path');
-import webpack = require('webpack');
-import packageJson = require('./package.json');
+const path = require('path');
+const webpack = require('webpack');
+const packageJson = require('./package.json');
```

Comment:
Could we change these back to` import`? The `eslint` complains "Require statement not part of import statement." if I use `const` instead.